CHANGELOG.md

```diff
@@ -1,9 +1,70 @@
+ * Fix numbering error in docs (Stefanie Molin, PR [16838](https://github.com/python/mypy/pull/16838))
+ * Various docs improvements (Shantanu, PR [16836](https://github.com/python/mypy/pull/16836))
+
+#### Stubtest Improvements
+  * Stubtest will ignore private function/method parameters when they are missing from the stub.
+Private parameters names start with a single underscore and have a default
+(PR [16507](https://github.com/python/mypy/pull/16507)).
+ * Stubtest: ignore a new protocol dunder (#16895) (Marc Mueller, PR [16899](https://github.com/python/mypy/pull/16899))
+ * stubtest: Private parameters can be omitted (Sebastian Rittau, PR [16507](https://github.com/python/mypy/pull/16507))
+ * stubtest: Add support for setting enum members to "..." (Jelle Zijlstra, PR [16807](https://github.com/python/mypy/pull/16807))
+ * stubtest: adjust symtable logic (Shantanu, PR [16823](https://github.com/python/mypy/pull/16823))
+ * stubtest: fix pos-only handling in overload resolution (Shantanu, PR [16750](https://github.com/python/mypy/pull/16750))
+
+#### Stubgen Improvements
+ * stubgen: Fix crash on star unpack of TypeVarTuple (Ali Hamdan, PR [16869](https://github.com/python/mypy/pull/16869))
+ * Fix failing stubgen tests (Ali Hamdan, PR [16779](https://github.com/python/mypy/pull/16779))
+ * stubgen: use PEP 604 unions everywhere (Ali Hamdan, PR [16519](https://github.com/python/mypy/pull/16519))
+ * Improve stubgen tests (Fabian Keller, PR [16760](https://github.com/python/mypy/pull/16760))
+ * stubgen: Do not ignore property deleter (Ali Hamdan, PR [16781](https://github.com/python/mypy/pull/16781))
+ * Support type stub generation for `staticmethod` (WeilerMarcel, PR [14934](https://github.com/python/mypy/pull/14934))
+
+#### Plugin Improvements
+ * Add `alias` support to `field()` in `attrs` plugin (Nikita Sobolev, PR [16610](https://github.com/python/mypy/pull/16610))
+ * Improve attrs hashability detection (Tin Tvrtković, PR [16556](https://github.com/python/mypy/pull/16556))
+
+#### PEP 696 Support & PEP 646 Improvements
+ * Expand TypeVarTuple default (PEP 696) (Marc Mueller, PR [16851](https://github.com/python/mypy/pull/16851))
+ * Fix TypeVar defaults with None (PEP 696) (Marc Mueller, PR [16859](https://github.com/python/mypy/pull/16859))
+ * Fix disallow-any errors for Instance types (PEP 696) (Marc Mueller, PR [16832](https://github.com/python/mypy/pull/16832))
+ * Apply TypeVar defaults to callables (PEP 696) (Marc Mueller, PR [16842](https://github.com/python/mypy/pull/16842))
+ * Use TypeVar defaults instead of Any when fixing TypeAlias types (PEP 696) (Marc Mueller, PR [16825](https://github.com/python/mypy/pull/16825))
+ * Use TypeVar defaults instead of Any when fixing instance types (PEP 696) (Marc Mueller, PR [16812](https://github.com/python/mypy/pull/16812))
+ * Consider TypeVarTuple to be invariant (Marc Mueller, PR [16759](https://github.com/python/mypy/pull/16759))
+
+#### Miscellaneous Fixes
```

Comment:
Consider combining `Plugin Improvements` and `Miscellaneous Fixes` and name it `Type-checking Improvements` similar to the notes for release 1.8.

and maybe further up, above below the new typechecking features.